### DISCUSSION TITLE
Trying to run silero-vad v4 on sonos/tract Rust library

### DISCUSSION THREAD

**Discussion by 7r3nzy** [snakers4/silero-vad#346] [Q&A] [answered]
There is an on going work on sonos/tract to support silero-vad v4 version, but we are running into some problems due to an `onnx IF operator`: 

https://github.com/sonos/tract/issues/1062

Could you guys help :)?

Thanks

**Comment by snakers4** [comment, marked as answer]
Hi,

If you would like us to adapt our model to your smart speaker, please reach out to us directly via email / telegram, we may discuss licensing options.

Best,
Alexander
